api/tests/v2/test_email.py

```diff
@@ -0,0 +1,127 @@
+from django.test import TestCase
+from rest_framework.test import APIRequestFactory, force_authenticate
+import mock
+
+from api.v2.views import (
+    VolumeSupportEmailViewSet, InstanceSupportEmailViewSet,
+    FeedbackEmailViewSet, ResourceEmailViewSet
+)
+from api.tests.factories import (
+    AnonymousUserFactory, InstanceFactory, UserFactory, VolumeFactory
+)
+
+
+class EmailTestHelper(TestCase):
```

Comment:
Small comment. If this class is just adding methods, you can express this intent in a subtle way by changing the name to `EmailTestMixin` and not having it inherit from `TestCase`. This is meaningful, because your class isn't trying to be something worth instantiating on its own, its just trying to mixin functionality.

Reply:
The reason I had it inherit from `TestCase` was so I could use `self.assertEquals`. I could change the name to `EmailTestMixin` to be more appropriate for its behavior as long as having a mixin inherit from another class is an acceptable practice